src/i18n/i18n.ts

```diff
@@ -20,7 +20,11 @@ export const availableLanguages = Object.keys(resources);
 i18n.use(LanguageDetector).use(initReactI18next).init({
   resources,
   fallbackLng: "en",
+
```

Comment:
Since we are passing an object in to `init` here, please remove the empty spaces and keep things tight

Reply:
https://github.com/reearth/reearth-web/pull/240/commits/7422aeaf7aa11eae174bacd235f1d5420f434829
I removed space 🙏